### PR TITLE
[pt-br] add docs/reference/glossary/persistent-volume-claim.md

### PR DESCRIPTION
**Pull request by akash-kumar-saw**
**Issue Number : #39524**

[pt-br] Localized en/docs/reference/glossary/persistent-volume-claim.md into Brazilian Portuguese

### CODE REVIEW

**Comment by akash-kumar-saw** [comment]
@MrErlison 
Please review this PR

File: content/pt-br/docs/reference/glossary/persistent-volume-claim.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Reivindicação de Volume Persistente
```
Comment (MrErlison) [inline content/pt-br/docs/reference/glossary/persistent-volume-claim.md:2]: I think it is better the original title, but I saw on the PersistentVolumes's doc "Requisição de Volume Persistente"

@stormqueen1990 What do you think about this?

File: content/pt-br/docs/reference/glossary/persistent-volume-claim.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+full_link: /docs/concepts/storage/persistent-volumes/#persistentvolumeclaims
+short_description: >
+  Declara recursos de armazenamento definidos em um PersistentVolume para que possa ser montado como um volume em um contêiner.
+
+aka: 
+tags:
+- core-object
+- storage
+---
+ Declara recursos de armazenamento definidos em um {{< glossary_tooltip text="PersistentVolume" term_id="persistent-volume" >}} para que possa ser montado como um volume em um {{< glossary_tooltip text="container" term_id="container" >}}.
```
Comment (MrErlison) [inline content/pt-br/docs/reference/glossary/persistent-volume-claim.md:14]: Minor: "contêiner" instead of "container"

**Comment by MrErlison** [comment]
Hello @akash-kumar-saw! Thanks for your contribution! 

Just a small fix.

File: content/pt-br/docs/reference/glossary/persistent-volume-claim.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Reivindicação de Volume Persistente
+id: persistent-volume-claim
+date: 2018-04-12
+full_link: /docs/concepts/storage/persistent-volumes/#persistentvolumeclaims
```
Comment (MrErlison) [inline content/pt-br/docs/reference/glossary/persistent-volume-claim.md:5]: Add - /pt-br/

File: content/pt-br/docs/reference/glossary/persistent-volume-claim.md (hunk 1 above)
Comment (akash-kumar-saw) [reply content/pt-br/docs/reference/glossary/persistent-volume-claim.md:2]: "Requisição de Volume Persistente" would be great... I am going ahead with "Requisição de Volume Persistente" as title.

**Comment by akash-kumar-saw** [comment]
Hi @MrErlison..
I have commited your suggestions :)
You can review it now...

**Comment by MrErlison** [comment]
Hi @akash-kumar-saw. The text looks great, but I'm going to wait for the reviewers to provide a second review.

File: content/pt-br/docs/reference/glossary/persistent-volume-claim.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+aka: 
+tags:
+- core-object
+- storage
+---
+ Declara recursos de armazenamento definidos em um {{< glossary_tooltip text="PersistentVolume" term_id="persistent-volume" >}} para que possa ser montado como um volume em um {{< glossary_tooltip text="contêiner" term_id="container" >}}.
+<!--more--> 
+
+Especifica a quantidade de armazenamento, como o armazenamento será acessado (somente leitura, leitura/gravação e/ou exclusivo) e como será recuperado (retido, reciclado ou excluído). Os detalhes do próprio armazenamento são descritos no objeto PersistentVolume.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/persistent-volume-claim.md:17]: Minor: I think "armazenamento propriamente dito" sounds better than "próprio armazenamento" in this last sentence.

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @akash-kumar-saw! 👋

Thank you very much for your contribution. 🎉

I found just one minor bit to be addressed. Other than that, it's a LGTM.

File: content/pt-br/docs/reference/glossary/persistent-volume-claim.md (hunk 4 above)
Comment (MrErlison) [reply content/pt-br/docs/reference/glossary/persistent-volume-claim.md:17]: +1

**Comment by akash-kumar-saw** [comment]
Hi @stormqueen1990,
I have commited to your suggestion.
You can review it now and can approve it :)

**Review by stormqueen1990 — COMMENTED** [review]
Thank you very much for your contribution, @akash-kumar-saw! 😃
/approve

**Pull request merged**